api-docs/swagger-v1.1.yml

```diff
@@ -1322,6 +1323,13 @@ parameters:
     in: query
     description: restrict by usage search query (name)
     required: false
+  usages_flat:
+    name: flat
+    type: boolean
+    in: query
+    description: format the returned set of usages - which by default is arranged in a tree structure - as a flat list; pagination options only apply if this option is true
+    default: false
+    required: false
```

Comment:
Dat is waar. Maar dat is eenvoudig toe te voegen zonder backwards incompatibilities. De huidige use case (autocompletion in een filter) lijkt me niet heel QM-specifiek, en daar is het endpoint wel al prima voor te gebruiken.